### PR TITLE
Gdr 2021

### PR DESCRIPTION
**Pull request by bczech**
# Description
## What changed?
Support for raw data in mixed formats (data.frame/data.table)
Related JIRA issue: GDR-2021

## Why was it changed?
To reprocess datasets using targets

# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [X] Package version bumped
- [X] Changelog updated

# Screenshots (optional)

### CODE REVIEW

File: R/prepare_input.R [hunk 1]
```diff
@@ -134,7 +134,7 @@ prepare_input.MultiAssayExperiment <-
       NULL
     })
     
-    if (!is.null(inl$df_)) {
+    if (length(inl$df_) > 0) {
```
Comment (marcin-kam) [inline R/prepare_input.R:137]: `> 0` not needed, it'll evaluate to `TRUE` for any length greater than 0

**Review by marcin-kam — APPROVED** [review]

**Review by gladkia — APPROVED** [review]

File: R/fit_SE.combinations.R [hunk 2]
```diff
@@ -231,7 +231,7 @@ fit_SE.combinations <- function(se,
         colnames(complete_subset), 
         c(norm_type, "row_values", "col_values", "codil_values")
       )
-      mat <- as.matrix(complete_subset[, ..keep])
+      mat <- as.matrix(complete_subset[, keep, with = FALSE])
```
Comment (DominikZabinski) [inline R/fit_SE.combinations.R:234]: How about `subset(complete_subset, select = keep)`?

Comment (bczech) [reply R/fit_SE.combinations.R:234]: For now, we use `[]` for subsetting in all the packages so I wouldn't mix different functions for the same thing

File: R/prepare_input.R (hunk 1 above)
Comment (bczech) [reply R/prepare_input.R:137]: Right

**Pull request merged**